.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,14 +4,15 @@
 
 **What is the purpose of this pull request? (put an "X" next to item)**
 
-[ ] Documentation update
-[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
-[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
-[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
-[ ] Add autofixing to a rule
-[ ] Add a CLI option
-[ ] Add something to the core
-[ ] Other, please explain:
+- [ ] Documentation update
+- [ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
+- [ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
+- [ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
+- [ ] Add autofixing to a rule
+- [ ] Add a CLI option
+- [ ] Add something to the core
+- [ ] Changes to build process
+- [ ] Other, please explain:
```

Comment:
Could you change these to complete the sentence “This PR will…” (i.e. “Fix a bug“, “Change the build process”)?